hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -146,7 +146,7 @@
 
   public static final String DFS_RATIS_SNAPSHOT_THRESHOLD_KEY =
       "dfs.ratis.snapshot.threshold";
-  public static final long DFS_RATIS_SNAPSHOT_THRESHOLD_DEFAULT = 10000;
+  public static final long DFS_RATIS_SNAPSHOT_THRESHOLD_DEFAULT = 100000;
```

Comment:
What's the purpose modified it?

Reply:
Hi @jiwq, thanks for the review. While doing performance testing with 160 client threads writing small files to a 50 node cluster continuously, we observed the snapshot taken a bit too often. Hence, this change to update the default limit with my change being in a very related area.